src/js/Controllers/DisplayCapabilities.js

```diff
@@ -174,11 +174,42 @@ let textbuttonsWithGraphicCapabilities = {
 	]
 }
 
+const buttonCapabilities = [{
+	"shortPressAvailable": true,
+	"longPressAvailable": true,
+	"upDownAvailable": true,
+	"name": "OK"
+},
+{
+	"shortPressAvailable": true,
+	"longPressAvailable": true,
+	"upDownAvailable": true,
+	"name": "PLAY_PAUSE"
+},
+{
+	"shortPressAvailable": true,
+	"longPressAvailable": true,
+	"upDownAvailable": true,
+	"name": "SEEKLEFT"
+},
+{
+	"shortPressAvailable": true,
+	"longPressAvailable": true,
+	"upDownAvailable": true,
+	"name": "SEEKRIGHT"
+},
+{
+	"shortPressAvailable": true,
+	"longPressAvailable": true,
+	"upDownAvailable": true,
+	"name": "CUSTOM_BUTTON"
+}]
+
 let capabilities = {
-	"TEXT_WITH_GRAPHIC": textWithGraphicCapabilities,
-	"GRAPHIC_WITH_TEXT": textWithGraphicCapabilities,
-	"TEXTBUTTONS_WITH_GRAPHIC": textbuttonsWithGraphicCapabilities,
-	"GRAPHIC_WITH_TEXTBUTTONS": textbuttonsWithGraphicCapabilities,
+	"TEXT_WITH_GRAPHIC": { ...textWithGraphicCapabilities, buttonCapabilities },
```

Comment:
These should definitely be included in the actual `textWithGraphicCapabilities` and `textbuttonsWithGraphicCapabilities` objects, rather than just tacked on here.